prog/dftb+/lib_common/constants.F90

```diff
@@ -110,8 +110,11 @@ module constants
   real(dp), parameter :: au__V_m = 1.0_dp / V_m__au
 
   !c3091013
+  !> Avagadros constant
+  real(dp), parameter :: avagadConst = 6.022140857e23_dp
```

Comment:
He is called "Avogardro" (https://en.wikipedia.org/wiki/Avogadro_constant), please change spelling :wink:  Also, please remove comment with user name.